db/changes.go

```diff
@@ -424,6 +424,7 @@ func (db *Database) SimpleMultiChangesFeed(chans base.Set, options ChangesOption
 			oldestSkipped := db.changeCache.getOldestSkippedSequence()
 			if oldestSkipped > 0 {
 				lowSequence = oldestSkipped - 1
+				base.Infof(base.KeyChanges, "%d is the oldest skipped sequence, using stable sequence number of %d for this feed", oldestSkipped, lowSequence)
```

Comment:
I realise it's slightly overkill to include both when they're always 1 apart, but:

* Makes it understandable to readers who might not be familiar with what's happening.
* Makes it nice for grepping - you'd probably be interested in this line whether you're chasing `1234` or `1235`:

```
1234 is the oldest skipped sequence, using stable sequence number of 1235 for this feed
```